go.mod

```diff
@@ -145,6 +146,8 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.4.0
 )
 
+replace github.com/pandatix/nvdapi => github.com/iansltx/nvdapi v0.0.0-20241125225022-a0b0fda988cb
```

Comment:
should we pull this into the fleetdm org?

Reply:
I think I have the maintainer's attention right now so my PR will get merged upstream soon enough. Setting myself a reminder to pull into our own repo in a week if it isn't upstream by then.